cmd/harmony/main.go

```diff
@@ -214,59 +219,103 @@ func passphraseForBls() {
 	blsPassphrase = passphrase
 }
 
-func setupInitialAccount() (isLeader bool) {
+func findAccountsByPubKeys(config shardingconfig.Instance, pubKeys []*bls.PublicKey) {
+	for _, key := range pubKeys {
+		_, account := config.FindAccount(key.SerializeToHexStr())
+		if account != nil {
+			initialAccounts = append(initialAccounts, account)
+		}
```

Comment:
let's printout if a key is not found for informational purpose.

Reply:
ditto

Reply:
`fmt.Printf("Bls key not found: %s\n", keyStr)`